src/annotator/guest.js

```diff
@@ -442,12 +444,14 @@ export default class Guest extends Delegator {
      */
     const highlight = anchor => {
       const range = resolveAnchor(anchor);
+      const colorList = ['#f0c1b3', '#acc5c8', '#93b8d3', '#eed8ae']
       if (!range) {
         return anchor;
       }
 
+		//also pass it the color to set it to, where get the color?
```

Comment:
Extra tab. Also, indentation in this line is using tabs instead of spaces (at least according to GitHub)